pkg/key/soft_key.go

```diff
@@ -0,0 +1,353 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package key
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/hex"
+	"errors"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/crypto"
+	"github.com/ava-labs/avalanchego/utils/formatting"
+	"github.com/ava-labs/avalanchego/utils/formatting/address"
+	"github.com/ava-labs/avalanchego/vms/components/avax"
+	"github.com/ava-labs/avalanchego/vms/platformvm"
+	"github.com/ava-labs/avalanchego/vms/secp256k1fx"
+	"go.uber.org/zap"
+)
+
+var (
+	ErrInvalidPrivateKey         = errors.New("invalid private key")
+	ErrInvalidPrivateKeyLen      = errors.New("invalid private key length (expect 64 bytes in hex)")
+	ErrInvalidPrivateKeyEnding   = errors.New("invalid private key ending")
+	ErrInvalidPrivateKeyEncoding = errors.New("invalid private key encoding")
+)
+
+var _ Key = &SoftKey{}
+
+type SoftKey struct {
+	privKey        *crypto.PrivateKeySECP256K1R
+	privKeyRaw     []byte
+	privKeyEncoded string
+
+	pAddr string
+
+	keyChain *secp256k1fx.Keychain
+}
+
+const (
+	privKeyEncPfx = "PrivateKey-"
+	privKeySize   = 64
+
+	rawEwoqPk      = "ewoqjP7PxY4yr3iLTpLisriqt94hdyDFNgchSxGGztUrTXtNN"
+	EwoqPrivateKey = "PrivateKey-" + rawEwoqPk
```

Comment:
There is a Prefix const somewhere, I think it's in avalanchego.
If not, you just created it above anyways

Reply:
fixed